javascript/0069-sqrtx.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Binary Search 
+ * https://leetcode.com/problems/sqrtx/
+ * 
+ * Time O(log(n)) | Space O(1)
+ * @param {number} x
+ * @return {number}
+ */
+var mySqrt = function(x) {
+    let left = 1;
+    let right = x;
+ 
+    while(left <= right) {
+        const mid = left + Math.floor((right - left)/2);
```

Comment:
Suggestion: Use Bitwise operator for shorter syntax

```js
const mid = ((left + right) >> 1);
```

Reply:
done.